keps/sig-network/1797-configure-fqdn-as-hostname-for-pods/kep.yaml

```diff
@@ -0,0 +1,48 @@
+title: Configure FQDN as Hostname for Pods
+kep-number: 1792
+authors:
+  - "@javidiaz"
+  - "@clrkio"
+  - "@kochut"
+owning-sig: sig-network
+participating-sigs:
+  - sig-network
+  - sig-node
+status: provisional
```

Comment:
I think you want to go to implementable TODAY if you can, right?

Reply:
yes, changed to implementable